kv/dist_sender.go

```diff
@@ -684,30 +687,29 @@ func (ds *DistSender) Send(_ context.Context, call client.Call) {
 		if finalReply != curReply {
 			// This was the second or later call in a multi-range request.
 			// Combine the new response with the existing one.
-			_, ok := finalReply.(proto.Combinable)
-			if !ok {
+			if cFinalReply, ok := finalReply.(proto.Combinable); ok {
+				cFinalReply.Combine(curReply)
+			} else {
```

Comment:
Go guidelines suggest the
```
cond := ...
if cond {
   return
}
work()
```

pattern unless the scope gets polluted, which isn't the case here. I prefer the original.